src/python/pants/init/options_initializer.py

```diff
@@ -149,10 +134,15 @@ def compute_pantsd_invalidation_globs(buildroot, bootstrap_options):
             )
         )
 
-        return list(invalidation_globs)
+        return tuple(x for x in invalidation_globs)
```

Comment:
This is creating an unnecessary generator.

```suggestion
        return tuple(invalidation_globs)
```